ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/security/security.component.ts

```diff
@@ -30,6 +30,8 @@ export class SecurityComponent implements OnInit {
     public formErrMessage = '';
     public inAdvancedMode = false;
     public signpostOpenState = false;
+    private tlsCaContents: any[] = [];
+    private registryCaContents: any[] = [];
     @Input() vchName: string;
 
     // tmp
```

Comment:
There used to be something else under `// tmp` which obviously makes that line obsolete now. Will remove it as well as empty ngOnInit()